ui/app/app.js

```diff
@@ -54,7 +54,7 @@ export default class App extends Application {
     },
     kv: {
       dependencies: {
-        services: ['download', 'router', 'store', 'secret-mount-path', 'flash-messages'],
+        services: ['download', 'namespace', 'router', 'store', 'secret-mount-path', 'flash-messages'],
```

Comment:
Just a general reminder that at least on any create or edit views, we need to surface the namespace reminder. e.g. you're doing such and such action inside a namespace.

Reply:
I think this also needs to be added to `ui/lib/kv/addon/engine.js`